docs/Server-Setup.md

```diff
@@ -249,4 +249,21 @@ The issue is documented in detail [here](https://github.com/benoitc/gunicorn/iss
 is to unselect `/tmp` in Docker's preferences under file sharing.
 
 The image below shows where the file sharing pane is.
-<img src="images/docker-preferences.png" />
\ No newline at end of file
+<img src="images/docker-preferences.png" />
+
+## Sending Slack notification in monitoring service
+
+If you need to send Slack notification from monitoring service, there are two ways to configure your system. 
+1. Using Slack rest API
+     * System prerequisite: install slackclient package on the platform where hosts monitoring service
+                `pip3 install slackclient`  
+     * Generate Slack API token via [create-api-token](https://slack.com/help/articles/215770388-create-and-regenerate-api-tokens)
+     * Figure out which channel you want to send notification to: SlACK_CHANNEL_ID
+     * Integrate your Slack messaging function to monitoring service: [Slack API basic_usage](https://slack.dev/python-slackclient/basic_usage.html)
+     * Start your monitoring service
+     * Note that this method doesn't have any restriction on your Slack plans.
+1. Using Slack Email App
```

Comment:
Give more context: say "System notifications (e.g., monitor.py) are sent to `$ADMIN_EMAIL`.  If you're using Slack, then you can set $ADMIN_EMAIL to a special email address which will show up in a designated slack channel.  See X for more details.  Note that this works only with the standard plan."